support-frontend/assets/pages/[countryGroupId]/student/components/StudentProductCard.tsx

```diff
@@ -0,0 +1,89 @@
+import { palette } from '@guardian/source/foundations';
+import {
+	LinkButton,
+	themeButtonReaderRevenueBrand,
+} from '@guardian/source/react-components';
+import { BenefitsCheckList } from 'components/checkoutBenefits/benefitsCheckList';
+import { simpleFormatAmount } from 'helpers/forms/checkouts';
+import { currencies } from 'helpers/internationalisation/currency';
+import { getBillingPeriodNoun } from 'helpers/productPrice/billingPeriods';
+import type { ThreeTierCardProps } from 'pages/supporter-plus-landing/components/threeTierCard';
+import { discountSummaryCopy } from 'pages/supporter-plus-landing/components/threeTierCard';
+import {
+	benefitsListCSS,
+	btnStyleOverrides,
+	container,
+	discountSummaryCss,
+	heading,
+	headWrapper,
+	originalPriceStrikeThrough,
+	pill,
+	promotionCss,
+} from './StudentProductCardStyles';
+
+export default function StudentProductCard({
+	cardContent,
+	promoCount,
+	currencyId,
+	billingPeriod,
+}: ThreeTierCardProps) {
```

Comment:
Would it be worth defining a `StudentProductCardProp` with just `cardContent` & `currencyId` to keep this interface as light a possible (ie hard code `cardTier`/`promoCount`/`isSubdued`/`BillingPeriod` inside of this component?) Is this what you meant by merging cards in future?